File: languageExtensions/OrchidSyntaxHighlighter/src/main/java/com/eden/orchid/languages/highlighter/HighlightResourceSource.java
package com.eden.orchid.languages.highlighter;

import com.eden.orchid.api.OrchidContext;
import com.eden.orchid.api.resources.resourceSource.PluginResourceSource;

import javax.inject.Inject;

public class HighlightResourceSource extends PluginResourceSource {

    @Inject
    public HighlightResourceSource(OrchidContext context) {
        super(context, 750);
    }

}

File: languageExtensions/OrchidSyntaxHighlighter/src/main/kotlin/com/eden/orchid/languages/highlighter/HighlightResourceSource.kt
package com.eden.orchid.languages.highlighter

import com.eden.orchid.api.OrchidContext
import com.eden.orchid.api.resources.resourceSource.PluginResourceSource

import javax.inject.Inject

class HighlightResourceSource @Inject
constructor(context: OrchidContext) : PluginResourceSource(context, 750)